### PR TITLE
conditional prompt if account is not linked

### PR DESCRIPTION
**Pull request by AndreEPaul**

### CODE REVIEW

File: front-end/src/components/Dashboard/index.js [hunk 1]
```diff
@@ -169,8 +183,13 @@ const DashboardUserContent = styled.div`
 `;
 
 const EditProfileLinkContainer = styled.div`
-  display: flex;
-  font-size: 12px;
+   display: flex;
+   flex-direction: column;
+   padding: 25px 25px 25px 25px;
```
Comment (jonabantao) [inline front-end/src/components/Dashboard/index.js]: If all sides are the same, you can just put padding: 25px

File: front-end/src/components/Dashboard/index.js [hunk 2]
```diff
@@ -74,47 +74,61 @@ class Dashboard extends React.Component {
   }
 
   renderDashboardContent = () => {
+    return <>
```
Comment (jonabantao) [inline front-end/src/components/Dashboard/index.js:77]: I think we should also have a placeholder image or their profile picture if their github info is attached

File: front-end/src/components/Dashboard/index.js (hunk 1 above)
Comment (AndreEPaul) [reply front-end/src/components/Dashboard/index.js]: fixed

**Review by Jedeu — APPROVED** [review]

File: front-end/src/components/Dashboard/index.js (hunk 2 above)
Comment (AndreEPaul) [reply front-end/src/components/Dashboard/index.js:77]: fixed. I'm using this which says it's free. I'm not sure the best practice for citing free images, do we need to? Here is the link: https://www.iconfinder.com/icons/2931147/account_avatar_face_head_human_profile_user_icon

**Review by mcmoonstruck — APPROVED** [review]

**Pull request merged**